docker/Dockerfile

```diff
@@ -0,0 +1,67 @@
+ARG CCF_VERSION="6.0.1"
+
+# Use fixed image version to ensure reproducibility
+ARG BASE_IMAGE=mcr.microsoft.com/azurelinux/base/core:3.0.20250402
+
+FROM ${BASE_IMAGE} AS builder
+
+ARG CCF_VERSION
+ARG CCF_PLATFORM
+ARG SCITT_VERSION_OVERRIDE
+
+RUN tdnf install -y ca-certificates && \
+    \
+    # Download the CCF development package
+    curl -L "https://github.com/microsoft/CCF/releases/download/ccf-${CCF_VERSION}/ccf_${CCF_PLATFORM}_devel_${CCF_VERSION//-/_}_x86_64.rpm" -o ccf.rpm && \
+    \
+    tdnf install -y ./ccf.rpm
+
+# Build CCF app
+COPY ./app /tmp/app/
+RUN mkdir /tmp/app-build && \
+    cd /tmp/app-build && \
+    CC=clang-18 CXX=clang++-18 cmake -GNinja \
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+    -DCMAKE_INSTALL_PREFIX=/usr/src/app \
+    -DCOMPILE_TARGET=${CCF_PLATFORM} \
+    -DBUILD_TESTS=OFF \
+    -DSCITT_VERSION_OVERRIDE=${SCITT_VERSION_OVERRIDE} \
+    /tmp/app && \
+    ninja && ninja install
+
+FROM ${BASE_IMAGE}
+
+ARG CCF_VERSION
+ARG CCF_PLATFORM
+
+WORKDIR /usr/src/app
+
+RUN tdnf install -y \
+    ca-certificates \
+    tini \
+    tar \
+    nc && \
```

Comment:
please use specific versions to aid in reproducibility

Reply:
Addressed in https://github.com/microsoft/scitt-ccf-ledger/pull/298/commits/bf22322a96a07fda8eae637247b494145222c10c